packages/insured-bridge-relayer/test/CrossDomainFinalizer.ts

```diff
@@ -0,0 +1,268 @@
+import {
```

Comment:
these tests are somewhat bloated. The reason is they'll need to consider the implementation of the `checkForFinalizedCanonicalRelaysAndFinalize` method, which will require all the L1 contracts deployed here.